src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -36,7 +36,12 @@
                       :currency="pointOfSalesCurrency"
                     />
                     <el-button slot="reference" type="text" style="color: #000000;font-weight: 604!important;font-size: 100%;">
-                      {{ formatPrice(currentOrder.grandTotal, pointOfSalesCurrency.iSOCode) }}
+                      {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes